admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -89,10 +94,14 @@ public ReplicationSiteField updateSite(String id, String name, AddressField addr
     return getSiteFieldForSite(site);
   }
 
-  private String addressFieldToUrlString(AddressField address) {
+  private String addressFieldToUrlString(AddressField address, @Nullable String rootContext) {
+    final String context =
+        (rootContext != null) ? rootContext.trim().replaceAll("^/*", "") : DEFAULT_CONTEXT;
```

Comment:
❓ Should this be replace all, or just the first one if it is the first character in the context path?

Reply:
Good point. This will do the right thing because of the "^" but it doesn't need the replace all.

Reply:
should we trim after the replace? or add use ( "^[/\s]*")

Reply:
Updated to do a trim first and then run the suggested regex